lightkurve/utils.py

```diff
@@ -305,7 +307,8 @@ def plot_image(image, ax=None, scale='linear', origin='lower',
         elif scale == 'sqrt':
             norm = ImageNormalize(vmin=vmin, vmax=vmax, stretch=SqrtStretch())
         elif scale == 'log':
-            norm = ImageNormalize(vmin=vmin, vmax=vmax, stretch=LogStretch())
+            norm = LogNorm(vmin=max(vmin, sys.float_info.epsilon), vmax=vmax,
```

Comment:
I would never have thought of this.  Smart!  Could you perhaps add a comment though to explain that you are doing this to avoid zero division?

Reply:
Yep! Done that! Does the comment look good?